app/javascript/application.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import App from './components/App';
+
+document.addEventListener('DOMContentLoaded', () => {
+  ReactDOM.render(
+    <Router>
+      <Switch>
```

Comment:
- [ ] [optional] @Rifat8080 I didn't understand the use of Switch, it led to the error below when trying to build the react app using webpack 
![image](https://github.com/Rifat8080/hello-rails/assets/61408860/bcdf54af-8193-40c2-a656-b5d73062ccc1)

`Kindly consider going through the tutorial again, I did fix the errors above and many others but still the app kept producing new errors all related to webpack`